hbase_sql/src/main/java/com/splicemachine/derby/impl/SpliceSpark.java

```diff
@@ -60,7 +60,8 @@
     // Sets both ctx and session
     public static synchronized SparkSession getSession() {
         String threadName = Thread.currentThread().getName();
-        if (!SpliceClient.isClient && !threadName.startsWith("olap-worker-")) {
+        if (!SpliceClient.isClient && !threadName.startsWith("olap-worker-")
+                && !threadName.startsWith("DRDAConnThread_")) { // stat collection job
              // Not running on the Olap Server... raise exception. Use getSessionUnsafe() if you know what you are doing.
```

Comment:
This was caused by removing StatsCollectionJob. You may consider to add it back.